libtsuba/src/RDGPartHeader.h

```diff
@@ -253,6 +270,53 @@ class KATANA_EXPORT RDGPartHeader {
   const PartitionMetadata& metadata() const { return metadata_; }
   void set_metadata(const PartitionMetadata& metadata) { metadata_ = metadata; }
 
+  uint32_t storage_format_version() const { return storage_format_version_; }
+  void update_storage_format_version() {
+    storage_format_version_ = latest_storage_format_version_;
+  }
+
+  const EntityTypeIDToSetOfEntityTypeIDsMap& node_entity_type_id_dictionary()
+      const {
+    return node_entity_type_id_dictionary_;
+  }
+
+  void set_node_entity_type_id_dictionary(
```

Comment:
These setters do not make sense to me. Who's calling these, as in what is the use case?

Reply:
the next layer up, RDG.h calls them
ex: `set_node_entity_type_id_name` in RDG.cpp
```
void
tsuba::RDG::set_node_entity_type_id_name(
    EntityTypeIDNamePairRange node_entity_type_id_name_range) {
  core_->part_header().set_node_entity_type_id_name(
      InflateEntityTypeIDNamePairRange(node_entity_type_id_name_range));
}
```